modules/core/src/weaver/Expectations.scala

```diff
@@ -6,6 +6,24 @@ import cats.data.{ NonEmptyList, Validated, ValidatedNel }
 import cats.effect.Sync
 import cats.syntax.all._
 
+trait Expectable[F[_], A] {
+  def lift(a: A): F[Expectations]
+}
+
+object Expectable {
+  def apply[F[_], B](implicit ev: Expectable[F, B]): Expectable[F, B] = ev
+  implicit def syncExpectable[F[_]: Sync]: Expectable[F, Expectations] =
```

Comment:
this should be constrained to `Applicative`, with `pure` being used

Reply:
love me a bit of Sync here and there. Good point, not sure why I needed this.